src/main/java/com/hazelcast/jdbc/JdbcResultSet.java

```
@@ -0,0 +1,1240 @@
+    public Date getDate(String columnLabel) throws SQLException {
+        return get(columnLabel);
+    }
+
+    @Override
+    public Time getTime(String columnLabel) throws SQLException {
+        return get(columnLabel);
+    }
+
+    @Override
+    public Timestamp getTimestamp(String columnLabel) throws SQLException {
+        return get(columnLabel);
+    }
+
+    @Override
+    public InputStream getAsciiStream(String columnLabel) throws SQLException {
+        checkClosed();
+        throw JdbcUtils.unsupported("Ascii Stream is not supported");
+    }
+
+    @Override
+    public InputStream getUnicodeStream(String columnLabel) throws SQLException {
+        checkClosed();
+        throw JdbcUtils.unsupported("Unicode Stream is not supported");
+    }
+
+    @Override
+    public InputStream getBinaryStream(String columnLabel) throws SQLException {
+        checkClosed();
+        throw JdbcUtils.unsupported("Binary Stream is not supported");
+    }
+
+    @Override
+    public SQLWarning getWarnings() throws SQLException {
+        checkClosed();
+        return null;
+    }
+
+    @Override
+    public void clearWarnings() throws SQLException {
+        checkClosed();
+    }
+
+    @Override
+    public String getCursorName() throws SQLException {
+        checkClosed();
+        throw JdbcUtils.unsupported("Cursor Name is not supported");
+    }
+
+    @Override
+    public ResultSetMetaData getMetaData() throws SQLException {
+        checkClosed();
+        throw JdbcUtils.unsupported("ResultSetMetaData not supported");
+    }
+
+    @Override
+    public Object getObject(int columnIndex) throws SQLException {
+        return get(columnIndex);
+    }
+
+    @Override
+    public Object getObject(String columnLabel) throws SQLException {
+        return get(columnLabel);
+    }
+
+    @Override
+    public int findColumn(String columnLabel) throws SQLException {
+        checkClosed();
+        return sqlResult.getRowMetadata().findColumn(columnLabel);
```

The method should throw an exception if the column is not found. In the current implementation, it will return -1.